staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service_unix_test.go

```diff
@@ -20,44 +20,54 @@ limitations under the License.
 package envelope
 
 import (
-	"context"
-	"encoding/base64"
 	"fmt"
-	"net"
-	"os"
 	"reflect"
-	"runtime"
-	"strings"
 	"sync"
 	"testing"
 	"time"
 
-	"google.golang.org/grpc"
+	mock "k8s.io/apiserver/pkg/storage/value/encrypt/envelope/testing"
 
 	"k8s.io/apimachinery/pkg/util/uuid"
-	kmsapi "k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1"
 )
 
+type testSocket struct {
+	path string
+	name string
```

Comment:
nit: can we call this `endpoint` to match the previous usage and the param name to NewGRPCService?

Reply:
still outstanding